InstallingDrivers.md

```diff
@@ -120,3 +120,78 @@ _Note: Only Skylake, Kaby Lake, and Coffee Lake offer full Vulkan support. Broad
 To install support for Vulkan API (will be functional only if you have a [Vulkan capable GPU](https://en.wikipedia.org/wiki/Vulkan_(API)#Compatibility) and driver), execute following command:
 
     sudo dnf install vulkan-loader vulkan-loader.i686
+
+    
+## openSUSE Tumbleweed/Leap
+
+First of all enabling multiarch or 32bit support is not required here it should not be disbaled in the first place.
+
+### AMD
+
+Usually if you run an AMD GPU with open source drivers the required packages should already be installed in case they don't run the following command:
```

Comment:
This needs some grammar fixes:
"Usually, if you have an AMD GPU running with open source drivers (Mesa), the required packages should already be installed. If for some reason they aren't, run the following command:"

Reply:
Will do